docs/WindowAggregation-sparkcompute.md

```diff
@@ -49,31 +53,80 @@ field called `nextValue` that is the next value of current row in the group. The
 **Number of partitions:** Number of partitions to use when grouping fields. If not specified, the execution
 framework will decide on the number to use.
 
+----------
+**Clause Constraints:**
+
+| Function | Partition fields | Order | Frame Type | 
+| :------------ | :------: | :----- | :---------- |
+| rank                    | required  | required      | not supported |
```

Comment:
Nitpick: Capitalize "Required" "Not required" and "Not supported". Function names in lowercase are fine.